spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'web_helpers'
+
+RSpec.feature "Sessions",  type: :feature do
+
+  scenario "signs up the user", :type => :feature do
+    sign_up
+    page.should have_content("Welcome! You have signed up successfully.")
+  end
+
+  scenario "signs in the user", :type => :feature do
+    sign_up
+    visit "/users/sign_out"
```

Comment:
`click_on("Sign Out")` instead of `visit "/users/sign_out"` should fix that :) 

When you `visit` sign out, you are sending a get request. But if you click on the 'sign out' button, you send a destroy request, which is the correct request.